datafusion/Cargo.toml

```diff
@@ -51,7 +51,7 @@ ahash = "0.7"
 hashbrown = { version = "0.11", features = ["raw"] }
 arrow = { version = "^5.2", features = ["prettyprint"] }
 parquet = { version = "^5.2", features = ["arrow"] }
-sqlparser = "0.9.0"
+sqlparser = "0.10.0"
```

Comment:
given that datafusion is used as a library, perhaps there is value in pinning the version with less restrictions say `0.10`, so downstream consumers can update the patch version if needed without running into version confilcts.

Reply:
```suggestion
sqlparser = "0.10"
```

Reply:
FWIW, I believe cargos solver / version scheme already allows this for `=` and `^=` (which are the same), regardless of the last `.0` included